documentation/projects/proposals/data_normalization/20240227-implementation_plan_catalog_data_cleaning.md

```diff
@@ -101,29 +99,44 @@ process][aws_mpu] will serve us here.
 | 2024-02-20 04:06:56 | 22157 |    9035456    |        8809209        |   0    |
 | 2024-02-13 04:41:22 | 22155 |    9035451    |        8809204        |   0    |
 
-To have some numbers of the problem we are delaing with, the previous table
+To have some numbers of the problem we are dealing with, the previous table
 shows the number of records cleaned by field for last runs at the moment of
 writing this IP, except for tags, which we don't have accurate registries since
 file saving was disabled.
 
+The alternative is to upload TSV files to the Amazon Simple Storage Service
+(S3), creating a new bucket or using a subfolder within `openverse-catalog`. The
+benefit of using S3 buckets is that they have streaming capabilities and will
+allow us to read the files in chunks later if necessary for performance. The
+downside is that objects in S3 don't allow appending natviely, so it may require
+to upload files with different part numbers or evaluate if the [multipart upload
+process][aws_mpu] or more easily, the `smart_open` package could serve us here.
```

Comment:
Do you mind linking `smart_open` to the library here as well?

Reply:
This was supposed to be the first link! 😆  Fixed.